ansible/roles/miniflux/defaults/main.yml

```diff
@@ -0,0 +1,307 @@
+# host. See the :ref:`debops.golang` role for more details.
+miniflux__binary: '{{ ansible_local.golang.binaries["miniflux"]
+                      if (ansible_local.golang.binaries|d() and
+                          ansible_local.golang.binaries.miniflux|d())
+                      else "" }}'
+
+                                                                   # ]]]
+                                                                   # ]]]
+# Configuration variables for other Ansible roles [[[
+# ---------------------------------------------------
+
+# .. envvar:: miniflux__postgresql__dependent_roles [[[
+#
+# Role configuration for the :ref:`debops.postgresql` Ansible role.
+miniflux__postgresql__dependent_roles:
+
+  - name: '{{ miniflux__database_user }}'
+
+  - name: '{{ miniflux__database_name }}'
+    flags: [ 'NOLOGIN' ]
+
+                                                                   # ]]]
+# .. envvar:: miniflux__postgresql__dependent_groups [[[
+#
+# Group configuration for the :ref:`debops.postgresql` Ansible role.
+miniflux__postgresql__dependent_groups:
+
+  - roles: [ '{{ miniflux__database_user }}' ]
+    groups: [ '{{ miniflux__database_name }}' ]
+    database: '{{ miniflux__database_name }}'
+
+                                                                   # ]]]
+# .. envvar:: miniflux__postgresql__dependent_databases [[[
+#
+# Database configuration for the :ref:`debops.postgresql` Ansible role.
+miniflux__postgresql__dependent_databases:
+
+  - name: '{{ miniflux__database_name }}'
+    owner: '{{ miniflux__database_name }}'
+
+                                                                   # ]]]
+# .. envvar:: miniflux__postgresql__dependent_pgpass [[[
+#
+# ``~/.pgpass`` configuration for the :ref:`debops.postgresql` Ansible
+# role.
+miniflux__postgresql__dependent_pgpass:
+
+  - owner: '{{ miniflux__user }}'
+    group: '{{ miniflux__group }}'
+    home: '{{ miniflux__home }}'
+    system: True
+
+                                                                   # ]]]
+# .. envvar:: miniflux__postgresql__dependent_extensions [[[
+#
+# Configuration of PostgreSQL extensions for the
+# :ref:`debops.postgresql` Ansible role.
+miniflux__postgresql__dependent_extensions:
+
+  - database: '{{ miniflux__database_name }}'
+    extension: 'hstore'
+
+                                                                   # ]]]
+# .. envvar:: miniflux__golang__dependent_packages [[[
+#
+# Configuration for the :ref:`debops.golang` Ansible role.
+miniflux__golang__dependent_packages:
+
+  - name: 'miniflux'
+    upstream_type: '{{ miniflux__upstream_type }}'
+    url:
+      - src: '{{ miniflux__upstream_url_mirror + "download/2.0.35/miniflux-" + miniflux__upstream_platform }}'
+        dest: 'releases/{{ miniflux__upstream_platform }}/miniflux/miniflux/2.0.35/miniflux_2.0.35_linux_amd64'
+    url_binaries:
+      - src: 'releases/{{ miniflux__upstream_platform }}/miniflux/miniflux/2.0.35/miniflux_2.0.35_linux_amd64'
```

Comment:
The version could be specified in a separate variable, `miniflux__upstream_version`, this could simplify upgrades. You can also add a watch file (you can find and example in the `docker_registry` role) and when you run the `make versions` command, the script will find out if upstream released a new version.